mod-audit-server/src/main/java/org/folio/dao/acquisition/impl/OrganizationEventsDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package org.folio.dao.acquisition.impl;
+
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Tuple;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.acquisition.OrganizationEventsDao;
+import org.folio.rest.jaxrs.model.OrganizationAuditEvent;
+import org.folio.rest.jaxrs.model.OrganizationAuditEventCollection;
+import org.folio.util.PostgresClientFactory;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.util.Date;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static org.folio.util.AuditEventDBConstants.ACTION_DATE_FIELD;
+import static org.folio.util.AuditEventDBConstants.ACTION_FIELD;
+import static org.folio.util.AuditEventDBConstants.EVENT_DATE_FIELD;
+import static org.folio.util.AuditEventDBConstants.ID_FIELD;
+import static org.folio.util.AuditEventDBConstants.MODIFIED_CONTENT_FIELD;
+import static org.folio.util.AuditEventDBConstants.ORDER_BY_PATTERN;
+import static org.folio.util.AuditEventDBConstants.ORGANIZATION_ID_FIELD;
+import static org.folio.util.AuditEventDBConstants.TOTAL_RECORDS_FIELD;
+import static org.folio.util.AuditEventDBConstants.USER_ID_FIELD;
+import static org.folio.util.DbUtils.formatDBTableName;
+
+@Repository
+public class OrganizationEventsDaoImpl implements OrganizationEventsDao {
+
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  public static final String TABLE_NAME = "acquisition_organization_log";
+
+  public static final String GET_BY_INVOICE_ID_SQL = "SELECT id, action, organization_id, user_id, event_date, action_date, modified_content_snapshot," +
+    " (SELECT count(*) AS total_records FROM %s WHERE organization_id = $1) FROM %s WHERE organization_id = $1 %s LIMIT $2 OFFSET $3";
+
+  public static final String INSERT_SQL = "INSERT INTO %s (id, action, organization_id, user_id, event_date, action_date, modified_content_snapshot)" +
+    " VALUES ($1, $2, $3, $4, $5, $6, $7)";
+
+  private final PostgresClientFactory pgClientFactory;
+
+  public OrganizationEventsDaoImpl(PostgresClientFactory pgClientFactory) {
+    this.pgClientFactory = pgClientFactory;
+  }
+
+  @Override
+  public Future<RowSet<Row>> save(OrganizationAuditEvent organizationAuditEvent, String tenantId) {
+    LOGGER.debug("save:: Saving Organization AuditEvent with tenant id : {}", tenantId);
+    String logTable = formatDBTableName(tenantId, TABLE_NAME);
+    String query = format(INSERT_SQL, logTable);
+    return makeSaveCall(query, organizationAuditEvent, tenantId)
+      .onSuccess(rows -> LOGGER.info("save:: Saved Organization AuditEvent with tenant id : {}", tenantId))
```

Comment:
more informative would be to log organizationId here instead of tenantId.
Also things like requestId, tenantId, userId should be automatically inserted by RMB at the beginning of log record.
Login of tenantId makes sense in case of ECS setup where we have processing in different tenants in the same time.